jina/parsers/hubble/push.py

```diff
@@ -31,13 +33,13 @@ def mixin_hub_push_parser(parser):
     mutually_exclusive_group.add_argument(
         '--public',
         action='store_true',
-        default=False,
+        default=argparse.SUPPRESS,
         help='If set, the published executor is visible to public',
     )
 
     mutually_exclusive_group.add_argument(
         '--private',
         action='store_true',
-        default=False,
+        default=argparse.SUPPRESS,
```

Comment:
```suggestion
```
i dont think it is needed

Reply:
@hanxiao `default=argparse.SUPPRESS` is used to skip the magic in `_get_help_string` in https://github.com/jina-ai/jina/blob/master/jina/parsers/helper.py#L112 

Without using `default=argparse.SUPPRESS`, the following help message (`--public` actually have default True to push a new executor) are displayed. 

```
--public    default: disabled, use "--public" to enable it
                  If set, the published executor is visible to public

--private   default: disabled, use "--private" to enable it
                  If set, the published executor is invisible to public
```

Reply:
i see, good point